onnx_tf/backend.py

```diff
@@ -671,6 +665,17 @@ def handle_conv(cls, node, input_dict):
   def handle_conv_transpose(cls, node, input_dict):
     return cls._conv(node, input_dict, transpose=True)
 
+  @classmethod
+  def handle_depth_to_space(cls, node, input_dict):
+    x = input_dict[node.inputs[0]]
+    x_rank = len(x.get_shape())
+    support_cuda = cls.supports_device("CUDA")
+    storage_format, compute_format = cls.get_data_format(x_rank, support_cuda)
+    x = tf.transpose(x, perm=cls.get_perm_from_formats(storage_format, compute_format))
```

Comment:
I think we should do the transpose iff storage_format differs from compute_format.

Reply:
@tjingrant 
So you mean if branch is desirable?

Reply:
Yeah.